src/Management/src/Endpoint/ManagementWebHostBuilderExtensions.cs

```diff
@@ -238,37 +238,20 @@ public static IWebHostBuilder AddThreadDumpActuator(this IWebHostBuilder builder
     }
 
     /// <summary>
-    /// Adds the Trace actuator to the application.
+    /// Adds the HttpExchange actuator to the application.
     /// </summary>
     /// <param name="builder">
     /// The <see cref="IWebHostBuilder" /> to configure.
     /// </param>
     /// <returns>
     /// The incoming <paramref name="builder" /> so that additional calls can be chained.
     /// </returns>
-    public static IWebHostBuilder AddTraceActuator(this IWebHostBuilder builder)
-    {
-        return AddTraceActuator(builder, MediaTypeVersion.V2);
-    }
-
-    /// <summary>
-    /// Adds the Trace actuator to the application.
-    /// </summary>
-    /// <param name="builder">
-    /// The <see cref="IWebHostBuilder" /> to configure.
-    /// </param>
-    /// <param name="mediaTypeVersion">
-    /// Specify the media type version to use in the response.
-    /// </param>
-    /// <returns>
-    /// The incoming <paramref name="builder" /> so that additional calls can be chained.
-    /// </returns>
-    public static IWebHostBuilder AddTraceActuator(this IWebHostBuilder builder, MediaTypeVersion mediaTypeVersion)
+    public static IWebHostBuilder AddHttpExchangesActuator(this IWebHostBuilder builder)
```

Comment:
I see both the names `HttpExchange` and `HttpExchanges` here. What's the correct terminology?